Editor/UI/EditorDrawingHelper.cs

```diff
@@ -64,6 +66,26 @@ public static bool DrawAddButton(ref Rect rect, string label)
             return GUI.Button(buttonRect, new GUIContent(label), style);
         }
 
+        public static bool DrawHelpButton(ref Rect rect)
+        {
+            rect.height = SingleLineHeight + addComponentButtonSize.y;
+
+            Rect addbuttonRect = rect;
```

Comment:
Duplicate code to DrawAddButton. Calculation of the button rects can be put in a separate method